Automattic/woocommerce-payments
# Fraud & Risk tools improvements from feedback

Fixes Automattic/woocommerce-payments-server#3204, Automattic/woocommerce-payments-server#3205, Automattic/woocommerce-payments-server#3206, Automattic/woocommerce-payments-server#3208

#### Changes proposed in this Pull Request

- Update banner discoverability copy.
- Add documentation links
- Update order note callout URL and description for NOT_WCPAY orders
- Update question mark icons' color









#### Testing instructions

**Pre-requisites**
* Checkout to this branch `fix/3181-demo-feedback`
* Update the version on `./woocommerce-payments.php` to be at least `5.7.0`
* Enable the fraud protection feature flag
  * If this is the first time, run `npm run wp option add wcpay_fraud_protection_settings_active 1`
  * If you already have it set, run `npm run wp option update wcpay_fraud_protection_settings_active 1`
* Build the client `npm run start`
* Listen to Stripe webhooks `npm run listen` – or equivalent
* Make sure you have another payment method as "Cash" configured

**Testing**
* Go to **Payments > Overview**
  * Read the discoverability banner text and click on "Learn more" – you should be redirected to the WooCommerce documentation
* Go to **Payments > Settings** and scroll to the **Fraud protection** section.
  * The question mark icons should be in the accent color (blue)
  * Check "Advanced" and click on "Configure"
  * Check all the links on the page to guarantee that they're linked to the right place
* Go to your shop and checkout an item paying with cash
* Go to **WooCommerce > Orders** and click on the latest order
* You should see the "Risk & Filter" meta box
<img width="293" alt="image" src="https://user-images.githubusercontent.com/16882226/229211544-58d0c37c-83be-43de-b28c-fb837a5e4673.png">






-------------------


**Post merge**
